pom.xml

```diff
@@ -49,7 +49,9 @@
       <tag>HEAD</tag>
   </scm>
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jenkins.version>1.609.2</jenkins.version>
+        <!-- TODO: Compilation errors to investigate when using 2.0 (htmlunit related) -->
+        <jenkins-test-harness.version>1.609.2</jenkins-test-harness.version>
```

Comment:
This code is already prepared, just not in `master`. We could backport most of [this](https://github.com/jenkinsci/workflow-plugin/compare/master...post-625).